cmd/iam.go

```diff
@@ -259,6 +259,32 @@ func (sys *IAMSys) LoadPolicy(objAPI ObjectLayer, policyName string) error {
 	return nil
 }
 
+// LoadPolicyMapping - loads the mapped policy for a user or group
+// from storage into server memory.
+func (sys *IAMSys) LoadPolicyMapping(objAPI ObjectLayer, userOrGroup string, isGroup bool) error {
+	if objAPI == nil {
+		return errInvalidArgument
+	}
+
+	sys.Lock()
+	defer sys.Unlock()
+
+	if globalEtcdClient == nil {
+		m := sys.iamUserPolicyMap
```

Comment:
Rather than like this can we make it safer by doing.. 

```
               if isGroup {
		   err := sys.store.loadMappedPolicy(userOrGroup, false, isGroup, sys.iamGroupPolicyMap)
               } else {
                   err := sys.store.loadMappedPolicy(userOrGroup, false, isGroup, sys.iamUserPolicyMap)
               }
```